test/e2e_env/kubernetes/testdata/externalservice-tcp-sink.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: externalservice-namespace
```

Comment:
something is wrong with namespaces.

You create `externalservice-namespace`, but I see in externalservice deployment that there is a separate namespace called `const externalServiceNamespace = DeploymentName + "namespace"` that is being managed by the deployment.

I might be wrong, but also I don't think you clean this namespace after the test.

Reply:
@jakubdyszkiewicz  

DeploymentName here is a constant `externalservice-` , not the name of the individual deployment. These are the namespaces that appear during the test:

```NAMESPACE                   NAME                                           READY   STATUS    RESTARTS   AGE
externalservice-namespace   externalservice-tcp-sink-7cdccf5c54-lfp4t      1/1     Running   0          51s
kube-system                 coredns-74ff55c5b-vzc7t                        1/1     Running   0          4m58s
kube-system                 etcd-kuma-1-control-plane                      1/1     Running   0          5m8s
kube-system                 kindnet-qjspd                                  1/1     Running   0          4m58s
kube-system                 kube-apiserver-kuma-1-control-plane            1/1     Running   0          5m8s
kube-system                 kube-controller-manager-kuma-1-control-plane   1/1     Running   0          5m8s
kube-system                 kube-proxy-k76t7                               1/1     Running   0          4m58s
kube-system                 kube-scheduler-kuma-1-control-plane            1/1     Running   0          5m8s
kuma-system                 kuma-control-plane-67c4fd4946-9dpgs            1/1     Running   0          86s
local-path-storage          local-path-provisioner-547f784dff-hh8pn        1/1     Running   0          4m58s
trafficlog-tcp-logging      demo-client-597d964bdc-pt67c                   2/2     Running   0          58s
trafficlog-tcp-logging      test-server-cd67cd488-xsqff                    2/2     Running   0          71s```

Reply:
Turns out the framework's externalservice deployment and associated namespace aren't as suited to this purpose in k8s, so moved the tcp sink into its own deployment.